bin/dbatools-index.json

```diff
@@ -47162,7 +47162,7 @@
                        ],
                        [
                            "InputObject",
-                           "Enables piped input of Microsoft.SqlServer.Management.Smo.Database, Microsoft.SqlServer.Management.Smo.Server, and Sqlcollaborative.Dbatools.Parameter.DbaInstanceParameter objects.",
+                           "Enables piped input of Microsoft.SqlServer.Management.Smo.Database, Microsoft.SqlServer.Management.Smo.Server, and dataplat.Dbatools.Parameter.DbaInstanceParameter objects.",

```

Comment:
hum...

Reply:
To update the class name we'd have to update it in the library everywhere.

Reply:
Thanks for the keen eye!

Reply:
closing this PR and starting over, brb